### PR TITLE
Update PyRAD to 3.0.66.

### PR DESCRIPTION
**Pull request by cbrueffer**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pyrad/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-  fn: 3.0.64.tar.gz
-  url: https://github.com/dereneaton/pyrad/archive/3.0.64.tar.gz
-  sha256: 45fd47819489df150427d5e28b8296b7aebae2281b8560412977668475c65796
+  fn: {{ version }}.tar.gz
+  url: https://github.com/dereneaton/pyrad/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
 
 build:
-  skip: True # [not py27]
+  skip: True  # [not py27]
```
Comment (mbargull) [inline recipes/pyrad/meta.yaml]: Can you add
```yaml
  entry_points:
    - pyrad = pyrad.pyRAD:main
```

File: recipes/pyrad/meta.yaml [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
 requirements:
   build:
```
Comment (mbargull) [inline recipes/pyrad/meta.yaml]: `build` requirements can be condensed to
```yaml
  build:
    - python
    - setuptools
```
. Also, I don't think six is needed as a `run` requirement.

File: recipes/pyrad/meta.yaml [hunk 3]
```diff
@@ -35,6 +39,6 @@ test:
     - pyrad -h
 
 about:
-  home: https://github.com/wichert/pyrad
+  home: https://github.com/dereneaton/pyrad
```
Comment (mbargull) [inline recipes/pyrad/meta.yaml]: Nice catch!

**Comment by cbrueffer** [comment]
Good comments, thanks @mbargull !

**Pull request merged**